proxy_wasm_api.h

```diff
@@ -473,6 +477,8 @@ class Context : public ContextBase {
 // Returns nullptr if the Context no longer exists (i.e. the stream has been
 // destroyed).
 Context *getContext(uint32_t context_id);
+RootContext *getRootContext(uint32_t context_id);
+ContextBase *getContextBase(uint32_t context_id);
```

Comment:
Nit: this seems unrelated.

Reply:
The getContextBase function is required in order to implement the new  EnvoyWasmVmIntegration::getNullVmFunction function in the envoy-wasm changes in order to support envoy_XXXX specific extensions to the ABI.

Reply:
getRootContext was added for consistency.